<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vOutputer</class>
 <widget class="QWidget" name="vOutputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>850</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelect">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbInfo1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>481</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbInfo2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>481</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbInfo3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>481</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblDate1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblDate2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblDate4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblDate3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbInfo4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>481</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSelect_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
